src/app/core/models/parser/roles.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Roles {
```

Comment:
Do you think we should indicate the possible roles, as such?
I think this helps speed up understanding for the reader, what do you think?
```
export interface Roles {
  'students'?: {
    [loginId: string]: string
  },
  'tutors'?: {
    [loginId: string]: string
  },
  'admins'?:{
    [loginId: string]: string
  }
}
```

Reply:
Oh yea true, I think that would be much clearer!